### ISSUE TITLE
Pumpjack for Oil not implemented?

### ISSUE THREAD

**Issue by Chaoschaot232** [KirkMcDonald/kirkmcdonald.github.io#192]
Not sure why this is all the time not fixed but when I want a setup for Oil Production, there is N/A for the pumpjack becuase it is no where in the source implemented, which is very very bad.

Also please add the latest 0.17.x and the latest exp. 0.18.x, thanks

**Comment by SquaredGrunt** [comment]
Pumpjacks production depends on the oil patch they are on, so it is impossible to define the number needed for a setup.

**Comment by Chaoschaot232** [comment]
> Pumpjacks production depends on the oil patch they are on, so it is impossible to define the number needed for a setup.

That isn't correct. It is true, that the oil patches are non-stable ressources but at there EOL you have a stable amount which you could use as default value.

In addition, you could add a field for oil patches value and do a bit of math if the PJs are running at all the time, to display after a) which time the patch is "depleted" and b) how this would change the production rate. With this you can then add the modules (speed, productivity, ...).

May a bit complicated but is is still possible ;)

**Comment by jxu** [comment]
Yep, assuming depleted (EOL) pumpjack is reasonable, giving 2 crude/s, with two speed 3 modules is 4 crude/s, then you can add on beacons if you wish.

Supposedly you could just add recipes to data/vanilla-1.1.19.json. I may try it out later
